omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,99 @@
+Using bulk imports
+==================
+
+The CLI import option ``--bulk`` specifies a configuration file that
+can be used to perform a batch of imports with the same or similar
+options. The file is written in a simple YAML syntax and can be placed
+anywhere.
+
+A simple YAML file might look like: ::
+
+    ---
+    continue: "true"
+    transfer: "ln_s"
+    path: "my-files.txt"
+
+Assuming that my-files.txt is a list of files such as ::
+
+    fileA
+    fileB
+    fileC
+
+this is equivalent to: ::
+
+    $ bin/omero import -k --transfer=ln_s fileA fileB fileC
+
+Bulk-only options
+-----------------
+
+Path
+^^^^
+
+The ``path`` key specifies a file from which *each individual line*
+will be processed as a separate import. In the simplest case, a single
+file is placed per line as above. For more complex usages, ``path``
+can point to a tab-separated value (TSV) file where each field will
+be interpreted based on ``columns``.
+
+Columns
+^^^^^^^
+
+A fairly regular requirement in importing many files is that for
+each file a similar but slightly different configuration is needed.
+This can be accomplished with the ``columns`` key. It specifies how
+each of the tab-separated fields of the ``path`` file (e.g. "my-files.txt")
+should be interpreted. For example, a bulk.yml file specifying::
+
+    path: "files.tsv"
+    columns:
+    - name
+    - path
+
+along with a files.tsv of the form::
```

Comment:
Question from audience during the Basel workshop:

   what is this mysterious .tsv format ? Do I have to create it in some special software ? Maybe worth mentioning ``table separated value``

Reply:
still think that csv will be more natural in the example

Reply:
sorry, missed https://github.com/openmicroscopy/ome-documentation/pull/1900/files#diff-d09750c6e463153813e3849d4ec6f4b8R35 - this is fine then